### ISSUE TITLE
JW Player Web 8.26.7 (8.27.0 tested as well) iOS 16.5 cropping responsive player

### ISSUE THREAD

**Issue by kylebradshaw** [jwplayer/jwplayer#4025]
### Expected Behavior

- with the newest iOS 16.5 update the embedded responsive player is cropped on it's edges
- before 16.5 the behavior works as intended

### Actual Behavior

<img width="505" alt="capture-2023-05-26-10 20 49@2x" src="https://github.com/jwplayer/jwplayer/assets/27273/1d15a461-5c95-4ffd-adb0-35797f1deab3">

- a _very_ fast orientation change and back _sometimes_ can fix the player
- going fullscreen also shows it stretched

<img width="579" alt="capture-2023-05-26-10 22 31@2x" src="https://github.com/jwplayer/jwplayer/assets/27273/a1ef4a5a-cb3b-49c9-bc81-9b879543d477">

this is the expected render

<img width="504" alt="capture-2023-05-26-10 24 02@2x" src="https://github.com/jwplayer/jwplayer/assets/27273/848db59c-abc3-4cf1-9568-dc10136026e5">

portions are significantly cropped

configuration is as follows:

```
export const configBase = (autostart = true) => ({
    aspectratio: '4:3',
    autostart,
    mute: true,
    primary: 'html5',
    hlshtml: true,
    width: '100%',
    repeat: true,
    liveTimeout: 0,
    responsive: true,
});
```

as you can see, significant cropping occurs

videos with more context

https://abiomed.box.com/s/ezx0t5p1t1uguww12opkr0vluzs25zls

### Steps to reproduce

1. load self-hosted player in auto-start fashion
2. with the newly released 16.5 I am seeing cropping for our streaming videos on impellaconnect.com 

### Environment

- iPhone 11pro, iPhone 12pro max
- iOS 16.5
- Mobile Safari 16.5?
- 8.26.7, 8.27.0

**Comment by kylebradshaw** [comment]
`stretching: 'fill'` in configuration fixed our issue

along with this style addition

```
.jwplayer.jw-stretch-uniform .jw-media video {
    object-fit: inherit !important;
}
````

**Comment by ntodd** [comment]
Observing this on iOS 16.5 as well. When testing the `stretching` config, below are the observed behaviors with the [documented options](https://docs.jwplayer.com/players/reference/setup-options).

- omitting `stretching` config: zooms as issue describes
- `uniform`: zooms as issue describes
- `exactfit`: stretches vertical or 4:3 videos
- `fill`: zooms vertical or 4:3 videos
-  `none`: plays video at native size, not desired

Oddly enough, when attempting to test `exactfit` I accidentally typed `fit`. This actually resolved the issue, no css changes required. It appears that including the `stretching` option, but setting it to anything but the above values (e.g. `foo`) actually corrects the issue, at least in my testing.

**Comment by kylebradshaw** [comment]
interesting, I noticed when you go fullscreen - same issue I'll try your "fix" to see if I get a similar outcome

are you on the latest lib? I think I'm locked in at 8.25.6 until they fix a `jw-text-elapsed` dvr bug I'm hitting as well

**Comment by ntodd** [comment]
Version 8.27.1 at the time of testing.

**Comment by zoeyim** [comment]
This has been fixed in v8.28.0: https://releases.jwplayer.com/web-8.28.0-JQLW8
